volto/src/addons/volto-govuk-theme/src/customizations/volto/components/theme/ServiceProblem/ServiceProblem.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Home container.
+ * @module components/theme/ServiceProblem/ServiceProblem
+ */
+
+import React from 'react';
+import { withServerErrorCode } from '@plone/volto/helpers/Utils/Utils';
+
+/**
+ * Service Problem function.
+ * @function ServiceProblem
+ * @returns {string} Markup of the not found page.
+ */
+const ServiceProblem = () => (
+  <div class="app-width-container">
+    <main
+      class="govuk-main-wrapper govuk-main-wrapper--l"
+      id="main-content"
+      role="main"
+    >
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-9">
+            Sorry, there is a problem with the service
+          </h1>
+          <p class="govuk-body">
```

Comment:
We should not merge/release Lorem ipsum code. I know this PR is still in draft and UCD mentioned this LI the other day, but if it comes to it and they can't settle what to put here, we're better off removing it than merging with this still in place.

Reply:
All good, removed it and replaced with some generic text, 'Our Support Team is working on the error, and will resolve it as soon as possible.'